x/coin_test.go

```diff
@@ -140,9 +97,9 @@ func TestValidCoin(t *testing.T) {
 		},
 		// make sure issuer is maintained throughout
 		{
-			NewCoin(2, -1500500500, "ABC").WithIssuer("my-chain"),
```

Comment:
Maybe adapt these tests more, not just remove.

I cannot find the changes to Compare... Can you link that to me? Needed for sort order. Or did it never consider issuer?

Reply:
Not sure what you're asking for. This is a test for a single `Coin` (not `Coins`)

I hope I have removed only those tests that were relying on the `Issuer` attribute.